libs/build_netcdf.sh

```diff
@@ -110,7 +110,7 @@ software=$name-"c"-$version
 [[ -d build ]] && rm -rf build
 mkdir -p build && cd build
 
-[[ ${STACK_netcdf_shared} =~ [yYtT] ]] || shared_flags="--disable-shared"
+[[ ${STACK_netcdf_shared} =~ [yYtT] ]] && shared_flags="--enable-shared --enable-static" || shared_flags="--disable-shared --enable-static"
```

Comment:
so are we building shared and static at all times?
If so, why not add that to the `configure` instead of duplicating `--enable-static` here for both if and else.

Reply:
see above comments, but all netcdf and hdf5 build script changes are reverted to noaa-emc/hpc-stack develop branch.